### PR TITLE
Feature/delete event

### PR DESCRIPTION
**Pull request by elijahbalo**

### CODE REVIEW

File: ionic/githubIonic/src/pages/list/list.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export class ListPage {
       afterFetch();
       this.events = [];
       response.forEach(eventFromBackend => {
-        this.events.push(new Event(eventFromBackend.title, null, null, null));
+        this.events.push(new Event(eventFromBackend.title, null, null, null,null));
```
Comment (kredods) [inline ionic/githubIonic/src/pages/list/list.ts:52]: why is the last one null ?

File: ionic/githubIonic/src/pages/profile/profile.html [hunk 2]
```diff
@@ -32,13 +32,24 @@ <h3 text-center>
...
+      <button ion-item  (click)="itemTapped(event)">
       <ion-icon name="wifi" item-left></ion-icon>
       {{event.title}}
       <div class="item-note" item-right>
         {{event.title}}
       </div>
-    </button>
+      </button>
+    
+
```
Comment (kredods) [inline ionic/githubIonic/src/pages/profile/profile.html]: empty space

File: ionic/githubIonic/src/pages/profile/profile.html [hunk 3]
```diff
@@ -32,13 +32,24 @@ <h3 text-center>
         Your Events
     </h3>
     <ion-list>
-        <button ion-item *ngFor="let event of events" (click)="itemTapped(event)">
+        <ion-item-sliding *ngFor="let event of events">
+
```
Comment (kredods) [inline ionic/githubIonic/src/pages/profile/profile.html]: more empty space

File: ionic/githubIonic/src/pages/profile/profile.html [hunk 4]
```diff
@@ -32,13 +32,24 @@ <h3 text-center>
...
+      </button>
+    
+
+
+    <ion-item-options side="right">
+      <button ion-button color="danger" (click)="deleteEvent(event.id)"><ion-icon name="trash"> </ion-icon></button>
+    </ion-item-options>
+  </ion-item-sliding>
+        
+
```
Comment (kredods) [inline ionic/githubIonic/src/pages/profile/profile.html]: same here

File: ionic/githubIonic/src/services/event.service.ts [hunk 5]
```diff
@@ -15,7 +15,9 @@ export class EventService {
   constructor(private http: Http, private authService: AuthService) { }
   // private instance variable to hold base url
   private Url = UrlProvider.url + '/api/events';
-
+
```
Comment (kredods) [inline ionic/githubIonic/src/services/event.service.ts]: wawu

File: ionic/githubIonic/src/pages/list/list.ts (hunk 1 above)
Comment (elijahbalo) [reply ionic/githubIonic/src/pages/list/list.ts:52]: shouldnt be null. ill change that

File: ionic/githubIonic/src/pages/profile/profile.html (hunk 2 above)
Comment (elijahbalo) [reply ionic/githubIonic/src/pages/profile/profile.html]: ill remove all empty spaces

**Review by kredods — APPROVED** [review]

**Pull request merged**